packages/webapp/README.md

```diff
@@ -2,27 +2,53 @@
 
 ## Getting started
 
-install dependencies
+1. Install dependencies
 ```
 pnpm i
 ```
 
-copy `.env.example` to `.env` and fill in `NEXT_PUBLIC_ALCHEMY_API_KEY`
+2. Copy `.env.example` to a new personal `.env` file and fill in ALL the missing values with your personal keys so we don't rate limit each other:
+```
+# RPC API KEYS
+NEXT_PUBLIC_ALCHEMY_API_KEY=YsaXbtaz1XXXXXXXXW2mADNtnHGLqwT
+
+# Block Explorer API KEYS
+NEXT_PUBLIC_ETHERSCAN_API_KEY=544JSDXXXXXXXXXXXXXXXXXXXXC7EUYMS
+NEXT_PUBLIC_BASESCAN_API_KEY=NMJ4FUEV3XXXXXXXXXXXXXXXXVBWWY28Z6
+
+NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=36fa892eXXXXX24c6dbf3bd572c9f
 
+# values are 'mainnet' or 'testnet'
+NEXT_PUBLIC_NETWORK_TYPE=mainnet
+```
 
-run dev server
+3. Run the dev server (automatically runs the predev as well to generate the latest schema so you are always up to date with any changes)
 ```
 pnpm dev
 ```
-
 ## Subgraph Queries
 
-This project uses [The Goldsky Client](https://goldsky.com/) to interact with the gnars subgraph in a type-safe way
+This project
+uses [The Goldsky Client](https://api.goldsky.com/api/public/project_clz4ukquribdy010b1fgua9nm/subgraphs/gnars-base/latest/gn)
+to interact with the Gnars subgraph in a type-safe way
 
-To add new queries, add them on the `queries` folder, then generate follow the README instructions in the subgraph
-package
+4. Add all new BASE queries/fragments/and requests to `packages/webapp/queries/base/**/*`.
 
+5. I modeled the new resolver for Gnars BASE after the Builder team's approach. You can look in their codebase for unlimited examples.  
+- After you have created a new Graphql , and you have tested it out on the Goldsky UI explorer, add it to the the queries directory and run `pnpm codegen` to update the new schema changes locally`.
+- Assuming `graphql-codegen` ran successfully, you should now be able to access it programmatically by `import`ing the `BaseSDK` client from `packages/webapp/queries/resolvers.ts`. It will create a singleton if connection to GoldSky if it doesn't already exist,  
```

Comment:
why put  "importing the `BaseSDK" in code block. also extra back quote messes with formatting

Reply:
BaseSDK is a variable name, I usually see people use `` syntax when quoting variables to add clarity

Reply:
"should be "singleton connection to Goldsky" (no "if")"
yes, fixing